modules/microshift-install-rpms.adoc

```diff
@@ -20,28 +20,26 @@ Use the following procedure to install {microshift-short} from an RPM package.
 [source,terminal,subs="attributes+"]
 ----
 $ sudo subscription-manager repos \
-    --enable rhocp-<4.18>-for-<9>-$(uname -m)-rpms \ # <1>
-    --enable fast-datapath-for-<9>-$(uname -m)-rpms # <2>
+    --enable rhocp-${ocp-version}-for-rhel-${op-system-version-major}-$(uname -m)-rpms \
+    --enable fast-datapath-for-rhel-${op-system-version-major}-$(uname -m)-rpms # <2>
```

Comment:
Need to remove `$` or it's not rendered correctly.
```suggestion
    --enable rhocp-{ocp-version}-for-rhel-{op-system-version-major}-$(uname -m)-rpms \
    --enable fast-datapath-for-rhel-{op-system-version-major}-$(uname -m)-rpms
```

Reply:
These `$` are not necessary for the commands?

Reply:
It renders:
$ sudo subscription-manager repos \
    --enable rhocp-$4.18-for-rhel-$9-$(uname -m)-rpms \
    --enable fast-datapath-for-rhel-$9-$(uname -m)-rpms (2)

Reply:
Depends where, not on the doc variables. It should render into the following:
```
$ sudo subscription-manager repos \
    --enable rhocp-4.18-for-rhel-9-$(uname -m)-rpms \
    --enable fast-datapath-for-rhel-9-$(uname -m)-rpms
```